test_scripts/API/Restructuring_OnResetTimeout/003_BC_OnResetTimeout_GENERIC_ERROR_in_case_HMI_did_not_respond.lua

```diff
@@ -0,0 +1,49 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0189-Restructuring-OnResetTimeout.md
+--
+-- Description:
+-- In case:
+-- 1) RPC is requested
+-- 2) Some time after receiving RPC request on HMI is passed
+-- 3) HMI sends BC.OnResetTimeout(resetPeriod = 7000) to SDL
```

Comment:
the description here `resetPeriod = 7000` does not match the code `resetPeriod = 9000`, please update

also it says `2) Some time after receiving RPC request on HMI is passed`, please mention that `some time` is actually 7 seconds, because that is in the code `notificationTime = 7000`